operator/README.md

```diff
@@ -231,46 +231,82 @@ $ make olm-uninstall
+See above for instructions on how to deploy an OLM bundle and index that was built locally.
 
-# Undeploy
-kubectl delete ns index-test
+### Prerequisites
+
+#### Required Binaries
+
+Both the `kubectl-kuttl` and `operator-sdk` binaries are required for the following make targets to work.
+There are make targets to install both executables:
+
+```bash
+make operator-sdk
+make kuttl
 ```
 
-## Extending the StackRox Custom Resource Definitions
+These make targets will add the executable to your `$GOPATH`.
+If that is not on your `$PATH`, then you can install the Operator SDK from its [release page](https://github.com/operator-framework/operator-sdk/releases)
+and kuttl from its [release page](https://github.com/kudobuilder/kuttl/releases/tag/v0.15.0).
 
-Instructions and best practices on how to extend the StackRox CRDs is contained in the separate file
-[EXTENDING_CRDS.md](./EXTENDING_CRDS.md).
+#### Pull Secret
 
-## Installing operator via OLM
+You'll also need a Quay pull secret configured in `~/.docker/config.json`.
+This can be retrieved on quay.io by:
+
+* Clicking on your profile in the top right corner
+* Choosing **Account Settings**
+* Under the **Docker CLI Password** section, click the **Generate Encrypted Password** link.
+* Enter your password and click **Verify**
+* Choose **Docker Configuration**
+* Either save the file to `~/.docker/config.json` or merge the quay config into your already-existing `config.json`.
+
+#### Clean Repo
+
+If `git describe --dirty` shows a `-dirty` suffix, you'll need to clean up your repo until git considers it "clean".
+Otherwise the make targets below will add `-dirty` to the image tag, and it likely won't be found.
 
-The following command will install operator to the currently selected kubernetes cluster.
+Note that if you run `olm-operator-install.sh` directly, this requirement does not apply.
+
+### Deploy
+
+Now the latest version (based off of `make tag`) can be installed like so:
 
 ```bash
- make kuttl deploy-via-olm
+# TODO(ROX-11744): drop branding here once operator is available from quay.io/stackrox-io
+
+ROX_PRODUCT_BRANDING=RHACS_BRANDING make deploy-via-olm
 ```
 
-If operator image has a `-dirty` suffix then the following command has to be used instead:
+This installs the operator into the `stackrox-operator` namespace.
+This can be overridden with the `TEST_NAMESPACE` argument:
 
 ```bash
-make kuttl deploy-dirty-tag-via-olm
+ROX_PRODUCT_BRANDING=RHACS_BRANDING make deploy-via-olm TEST_NAMESPACE=my-favorite-namespace
 ```
 
-For upgrading an existing operator:
+If you'd rather put in a custom image spec, you can use the install script directly:
 
 ```bash
-make kuttl upgrade-via-olm
+hack/olm-operator-install.sh stackrox-operator quay.io/rhacs-eng/stackrox-operator 3.74.0-588-ge99fe7b316
+```
+
+### Removal
 
+You can blow everything away with:
+
+```bash
+$ make olm-uninstall
+$ kubectl delete ns stackrox-operator
```

Comment:
Consider also running `make uninstall` to remove the CRDs from the cluster.

Reply:
Thanks, added.